error: 'Init' is not a member of 'IfcParse::IfcFile' when compiling with USE_MMAP

Hi there,

I am trying to compile the current head of the v0.6.0 branch in a Dockerfile, but I can't seem to get the `USE_MMAP` to work. I get the following error:

```sh
/IfcOpenShell/src/ifcparse/IfcParse.cpp: In constructor 'IfcParse::IfcFile::IfcFile(const string&, bool)':
/IfcOpenShell/src/ifcparse/IfcParse.cpp:1307:18: error: 'Init' is not a member of 'IfcParse::IfcFile'
 1307 |  return IfcFile::Init(new IfcSpfStream(fn, mmap));
      |                  ^~~~
/IfcOpenShell/src/ifcparse/IfcParse.cpp:1307:49: error: returning a value from a constructor
 1307 |  return IfcFile::Init(new IfcSpfStream(fn, mmap));
      |                                                 ^
make[2]: *** [CMakeFiles/IfcParse.dir/build.make:245: CMakeFiles/IfcParse.dir/IfcOpenShell/src/ifcparse/IfcParse.cpp.o] Error 1
```

Dockerfile:
```dockerfile
FROM ubuntu:20.10 as builder

RUN apt-get update && apt-get install --yes git
RUN git clone https://github.com/IfcOpenShell/IfcOpenShell.git

RUN cd IfcOpenShell && git checkout e1fa6fb3
RUN apt-get install --yes cmake g++ libboost-all-dev
RUN apt-get install --yes liboce-foundation-dev \
    liboce-modeling-dev liboce-ocaf-dev liboce-visualization-dev \
    liboce-ocaf-lite-dev
RUN cd IfcOpenShell && mkdir build && cd build && \
    cmake ../cmake -DOCC_LIBRARY_DIR=/usr/lib/x86_64-linux-gnu/ -DCOLLADA_SUPPORT=0 -DIFCXML_SUPPORT=0 -DBUILD_IFCGEOM=0 \
    -DBUILD_IFCPYTHON=0 -DBUILD_EXAMPLES=0 -DBUILD_GEOMSERVER=0 -DUSE_MMAP=1
# Wish we could use MMAP but it triggers a bug witnh the current git commit
RUN cd IfcOpenShell/build/ && make
```

--- a/src/ifcparse/IfcParse.cpp
+++ b/src/ifcparse/IfcParse.cpp
@@ -1304,7 +1304,7 @@ void IfcEntityInstanceData::setArgument(unsigned int i, Argument* a, IfcUtil::Ar
 //
 #ifdef USE_MMAP
 IfcFile::IfcFile(const std::string& fn, bool mmap) {
-	return IfcFile::Init(new IfcSpfStream(fn, mmap));
+	initialize_(new IfcSpfStream(fn, mmap));
 }
 #else
 IfcFile::IfcFile(const std::string& fn) {

